content/client-server-api/modules/end_to_end_encryption.md

```diff
@@ -597,6 +597,30 @@ package naming convention.
 
 {{% event event="m.key.verification.cancel" %}}
 
+###### General key verification error handling
+
+At any point the interactive verification can go wrong. The following describes
+what to do when an error happens:
+
+-   Alice or Bob can cancel the verification at any time. An
+    `m.key.verification.cancel` message must be sent to signify the
+    cancellation.
+-   The verification can time out. Clients should time out a verification that
+    does not complete within 10 minutes. Additionally, clients should expire a
+    `transaction_id` which goes unused for 10 minutes after having last
+    sent/received it. The client should inform the user that the verification
+    timed out, and send an appropriate `m.key.verification.cancel` message to
+    the other device.
+-   When the same device attempts to initiate multiple verification attempts,
+    the recipient should cancel all attempts with that device.
+-   When a device receives an unknown `transaction_id`, it should send an
+    appropriate `m.key.verification.cancel` message to the other device
+    indicating as such. This does not apply for inbound
+    `m.key.verification.request` or `m.key.verification.cancel` messages.
```

Comment:
We should probably fix this documentation, then

![image](https://github.com/matrix-org/matrix-spec/assets/16734772/e58c75d6-32ab-4b10-89f6-d8c4192590f8)

Since the `m.relates_to` is not required if sending the start as the first message, right?

And the `transaction_id` is just arbitrary if not related to a request.

Reply:
https://github.com/matrix-org/matrix-spec/pull/1830/commits/4282a1a319362cf8dca6948ef77b1a6b3a464a27

Reply:
You can only initiate the verification with a `.start` if it's done using to-device messages.  (Confusing, I know.)  So the "Required when sent as an in-room message" for `m.relates_to` is still correct.